### DISCUSSION TITLE
Retrieve abandoned username

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22643] [New to GitHub]
<p>I wanted to have the username of my project on this account but I found that there’s an account with that username that has no activities since the “Joined GitHub” in 2014, no contribution or stars in the past 5 years. Github have some policy regarding abandoned accounts? There’s anything I could do to retrieve it?</p>

**Comment by Li-Silke** [comment, marked as answer]
<p>Hello <a class="mention" href="/kornolinka">@kornolinka</a> ,</p>
<p>Thanks for reaching out! There is a <a href="https://help.github.com/articles/name-squatting-policy/" rel="nofollow noopener">policy on name squatting</a> which allows people to acquire accounts without activity. You’ll need to reach out to <a href="https://github.com/contact" rel="nofollow noopener">GitHub Support</a> directly for help with this. We can’t discuss specific account details in the public forum.</p>
<p>I hope this helps.</p>

**Comment by samus-aran** [comment]
Hi everyone 👋 ,

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
